Common/UnitDefinitions/InverseAngle.json

```diff
@@ -0,0 +1,39 @@
+{
+  "Name": "InverseAngle",
```

Comment:
What are some typical use cases for this quantity?

Just trying to make sure the quantities are widely used and avoid bloat, we have a small issue with binary size. I could not find much references by a quick google.

Reply:
Probably not very widely used, but a lot of our stability and control derivatives are in inverse degrees or inverse radians.

Reply:
Since we have 100+ quantities and 1000+ units, I am hesitant to start adding the inverse variant of all our quantities and units. 

We are adding #954 , but that one does seem like a widely used quantity with both wiki article and online converters for.

Are you OK with closing this PR, or do you still see value for others in adding inverse angle?